### PR TITLE
Pic 498 add licence conditions endpoint

### PR DESCRIPTION
**Pull request by jhevans**

### CODE REVIEW

**Review by chris-faulkner-mojdt — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Disposal.java [hunk 1]
```diff
@@ -83,6 +89,9 @@
     @Column(name = "ENTERED_NOTIONAL_END_DATE")
     private LocalDate enteredSentenceEndDate;
 
+    @OneToMany(targetEntity = LicenceCondition.class, mappedBy = "disposal")
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Disposal.java:92]: do you need `targetEntity = LicenceCondition.class` ?

File: src/test/java/uk/gov/justice/digital/delius/service/RequirementServiceTest.java [hunk 2]
```diff
@@ -62,7 +67,81 @@ public void setUp() {
...
+                .commencementDate(commencementDate)
+                .commencementNotes("Commencement notes")
+                .startDate(startDate)
+                .terminationDate(terminatedDate)
+                .terminationNotes("Termination notes")
+                .createdDateTime(createdDateTime)
+                .activeFlag(1L)
+                .build()));
+
+        LicenceConditions conditions = requirementService.getLicenceConditionsByConvictionId(CRN, CONVICTION_ID);
```
Comment (andymarke) [inline src/test/java/uk/gov/justice/digital/delius/service/RequirementServiceTest.java:91]: maybe use `var` in all these varaibles?

**Review by andymarke — APPROVED** [review]
LGTM

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**